api/README.md

```diff
@@ -1283,3 +1286,75 @@ Content-Type: application/json
   { "id": "id3", "doc": { <...the hydrated document...> } },
 ]
 ```
+
+
+# Contacts by phone
+
+Accepts a phone number parameter and returns fully hydrated contacts that match the requested phone number.
+If multiple contacts are found, all are returned.  When no matches are found, a 404 error is returned.
+Supports both GET and POST. 
+Only allowed for users with "online" roles. 
+
+## GET api/v1/contacts-by-phone
+
+#### Query parameters
+
+| Name | Required | Description |   
+| -----  | -------- | ------ | 
+| phone | true | A URL encoded string representing a phone number |
+
+ 
+#### Example
+
+```
+GET /api/v1/contacts-by-phone?phone="%2B40+(744)+999+999"
```

Comment:
What do the `+` represent here? If they are spaces we may want to use `%20` instead of `+` to avoid confusion.

Reply:
The `+` are spaces yes. but actual spaces: "%2B40+(744)+999+999" = "+40 (744) 999 999" decoded.
It's just to exemplify that this is a **URL Encoded** string.

Reply:
Hmm, I am a bit confused by the use of the `+` here. 

Using urldecoder.org I see the following:
`%2B40+(744)+999+999` = `+40+(744)+999+999` decoded.

![image](https://user-images.githubusercontent.com/789512/83432961-363e4b80-a407-11ea-926f-5bb940cd5150.png)

Do you know what I am doing wrong with the decoding? 

Showing `+` and `%2B` may be confusing, could we use the full URL encoded version eg `%2B40%20%28744%29%20999%20999`, or with dashes `%2B40-44-999-999`?

Reply:
@abbyad I've ported the documentation part over to cht-docs: https://github.com/medic/cht-docs/pull/154 and included your suggestion.